test/sender.js

```diff
@@ -207,13 +223,73 @@ describe('Event sender (secure)', () => {
   })
 })
 
+describe('Event sender (RELP)', () => {
+
+  let server;
+
+  beforeEach(async() => {
+    server = await new TestServer(relpOptions)
+  });
+
+  afterEach(async() => {
+    await server.close();
+  });
+
+  it('sends many events', done => {
+    const sender = senderLib.create(relpOptions)
+    const txnos = new Set();
+    txnos.add(1); // open uses txno 1
+    sender.on('error', done)
+    for (let i = 0; i < 100; i++)
+      txnos.add(sender.send(messageString));
+    txnos.add(sender.sendClose());
+    txnos.size.should.be.exactly(102);
+    let openRsp = false;
+    let closeRsp = false;
+    sender.on('rsp', rsp => {
+      txnos.delete(rsp.txno);
+      openRsp = openRsp || rsp.command === 'open';
+      closeRsp = closeRsp || rsp.command === 'close';
+      if(txnos.size === 0) {
+        openRsp.should.be.true();
+        closeRsp.should.be.true();
+        done();
+      }
+    });
+  });
+
+  it('resend', done => {
+    const sender = senderLib.create(relpOptions)
+    const txno = sender.send(messageString);
+    for (let i = 0; i < 99; i++)
+      sender.resend(messageString, txno).should.be.exactly(txno);
+    sender.sendClose();
+    let txnoRsps = 0;
+    let openRsp = false;
+    let closeRsp = false;
+    sender.on('rsp', rsp => {
+      if(rsp.txno === txno) txnoRsps++;
+      openRsp = openRsp || rsp.command === 'open';
+      closeRsp = closeRsp || rsp.command === 'close';
+      if(txnoRsps === 100 && openRsp && closeRsp) done();
+    });
+  });
+
+})
```

Comment:
shoudl we add a rule to the linter in order to identify missing `;`s?

Reply:
Totally agree. I updated the linter rules with the ones we use to enable. And there were a ton of missing semicolons.

c467f5e0e985979790e0188cd07ae073d0c5e5d5